samples/cpp/benchmark_app/main.cpp

```diff
@@ -539,6 +539,12 @@ int main(int argc, char* argv[]) {
 
         // If set batch size, disable the auto batching
         if (FLAGS_b > 0) {
+            if ((!FLAGS_hint.empty() && (FLAGS_hint == "throughput" || FLAGS_hint == "tput" ||
+                                         FLAGS_hint == "CUMULATIVE_THROUGHPUT" || FLAGS_hint == "ctput")) ||
```

Comment:
Why do we have such mess with low and high case?

Reply:
have change "CUMULATIVE_THROUGHPUT" to "cumulative_throughput"